lxd/apparmor/archive.go

```diff
@@ -24,7 +24,7 @@ profile "{{.name}}" {
   {{ .backupsPath }}/** rw,
   {{ .imagesPath }}/** r,
 
-  signal (receive) set=("term") peer=unconfined,
+  signal (receive) set=("term"),
```

Comment:
Is this because LXD is now not running under "unconfined" but instead running in a confined profile that is effective unconfined (hence the "(unconfined)" in the name, and so we need to relax the check here to allow signals from confined processes?

Reply:
Yes that is correct - I'll update the comment as you requested for the other code section which will explain this.